makefile

```diff
@@ -799,7 +799,8 @@ ifeq ($(kernel),darwin)
 		sdk-dir = $(platform-dir)/Developer/SDKs
 
 		ios-version := $(shell \
-				if test -L $(sdk-dir)/$(target)9.1.sdk; then echo 9.1; \
+				if test -L $(sdk-dir)/$(target)9.2.sdk; then echo 9.2; \
```

Comment:
This is getting slightly out of hand.  How about something like this:?

`for x in 9.3 9.2 9.1; do if test -d $(sdk-dir)/$(target)$x.sdk -o -L $(sdk-dir)/$(target)$x.sdk; then echo $x; break; fi; done`